packages/sources/src/styling/global/footer.scss

```diff
@@ -16,41 +17,41 @@ footer {
   }
 
   h2 {
-    border-bottom: 1px solid $dso-footer-border-color;
+    border-bottom: 1px solid $dso-footer-header-border-color;
+    color: $grijs-90;
     font-size: 1.25em;
     font-weight: 700;
-    margin-bottom: $u1;
-    margin-top: 0;
-    padding-bottom: $u1;
+    margin-bottom: $u2;
+    margin-top: $u2;
+    padding-bottom: $u2;
   }
 
   a {
-    @include anchor-reverse();
-
-    color: $bosgroen;
-
+    &,
     &:hover,
     &.active,
-    &:active {
+    &:active,
+    &:focus {
       color: $grijs-90;
     }
 
-    &:visited {
-      color: $bosgroen;
-
+    &:not(.btn):visited {
+      &,
       &:hover,
       &.active,
-      &:active {
+      &:active,
+      &:focus {
         color: $grijs-90;
       }
     }
   }
 
-  ul:not(.dso-link-list) {
-    @include list-unstyled();
+  a:hover,
+  a:focus {
+    text-decoration: none;
 
-    li {
-      line-height: 2em;
+    li & {
```

Comment:
dit doen we zo min mogelijk, omdat styling ook in sommige projecten IN een andere scss kan worden geincluded (en daar krijg je dan potentieel hele diepe/rare css van).

Reply:
AH ja, dat had ik eerst maar ben een beetje doorgeslagen met code cleanen ;)

Reply:
Ok aanpassen maar...